deployments/helm/nsm/templates/nsm-configmap.yaml

```diff
@@ -0,0 +1 @@
+../../../../k8s/conf/nsm-configmap.yaml
```

Comment:
Hmm? Are you sure?

Reply:
well.. the symlink is intended, to avoid duplication, if that is what the question about

Reply:
The symlink is not a good practice. Please reconsider how this can be avoided.

Reply:
@edwarnicke @haiodo I'd like to see your opinion here. Is this something we want to keep?